Dockerfile

```diff
@@ -1,12 +1,14 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
+MAINTAINER Aleksandar Todorović <aleksandar@r3bl.me>
```

Comment:
I wouldn't think this makes sense since this is maintained by the mailpile community